src/components/about-chat-sidebar/AboutChatSidebar.tsx

```diff
@@ -0,0 +1,144 @@
+import { FC, useState } from 'react'
+import { useNavigate } from 'react-router-dom'
+import { useTranslation } from 'react-i18next'
+
+import Typography from '@mui/material/Typography'
+import Box from '@mui/material/Box'
+import Divider from '@mui/material/Divider'
+import Avatar from '@mui/material/Avatar'
+
+import IconButton from '@mui/material/IconButton'
+import ImageOutlinedIcon from '@mui/icons-material/ImageOutlined'
+import InsertDriveFileOutlinedIcon from '@mui/icons-material/InsertDriveFileOutlined'
+import LinkOutlinedIcon from '@mui/icons-material/LinkOutlined'
+import Close from '@mui/icons-material/Close'
+
+import AppButton from '~/components/app-button/AppButton'
+import SidebarContentBox from '~/components/sidebar-content-box/SidebarContentBox'
+import SidebarImageGrid from '~/components/sidebar-image-grid/SidebarImageGrid'
+import FileComponent from '~/components/file-component/FileComponent'
+import LinkComponent from '~/components/link-component/LinkComponent'
+
+import { SizeEnum, ButtonVariantEnum, UserResponse, Link, File } from '~/types'
+import { createUrlPath, spliceSx } from '~/utils/helper-functions'
+import { authRoutes } from '~/router/constants/authRoutes'
+
+import { styles } from '~/components/about-chat-sidebar/AboutChatSidebar.styles'
+
+interface AboutChatSidebarProps {
+  user: UserResponse
+  media: Array<string>
+  files: File[]
+  links: Link[]
+}
+
+const AboutChatSidebar: FC<AboutChatSidebarProps> = ({
+  user,
+  media,
+  files,
+  links
+}) => {
+  const { t } = useTranslation()
+  const navigate = useNavigate()
+  const [isOpened, setIsOpened] = useState(true)
+
+  const {
+    _id,
+    firstName,
+    lastName,
+    photo,
+    role,
+    professionalSummary: userDescription
+  } = user
+
+  const userProfileURL = createUrlPath(authRoutes.userProfile.path, _id, {
+    role
+  })
+
+  const closeSidebar = () => {
+    setIsOpened(false)
+  }
+
+  return (
+    <Box
+      data-testid='sidebar'
+      style={isOpened ? undefined : { display: 'none' }}
```

Comment:
You can pass "isOpened" to styles.wrapper(isOpened) and move this check to styles for example: 
wrapper: (isOpened: boolean) => ({ 
display: isOpened : 'block' :  'none'
})

Reply:
> You can pass "isOpened" to styles.wrapper(isOpened) and move this check to styles for example: wrapper: (isOpened: boolean) => ({ display: isOpened : 'block' : 'none' })

Wow, that's actually a much better solution. Thank you for your advice, fixed.